enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/InternalFlatFileRealm.java

```diff
@@ -212,26 +212,17 @@ private void readFilesFromDisk( int attemptLeft, java.util.List<String> failures
         {
             if ( userRepository.numberOfUsers() == 0 )
             {
-                if ( initialUserRepository.numberOfUsers() == 0 )
+                User neo4j = newUser( INITIAL_USER_NAME, "neo4j", true );
```

Comment:
No, we always want the initial user. The next check is only to see if the initial password has been set or not.